islandora.module

```diff
@@ -251,7 +251,8 @@ function islandora_jsonld_alter_normalized_array(EntityInterface $entity, array
     $reaction->execute($entity, $normalized, $context);
     foreach ($context_manager->getActiveContexts() as $context_config) {
       try {
-        if ($context_config->getReaction($reaction->getPluginId())) {
+        if ($context_config->getReaction($reaction->getPluginId()) && isset($context['cacheability'])
+        && method_exists($context['cacheability'], 'addCacheTags')) {
           $context['cacheability']->addCacheTags($context_config->getCacheTags());
```

Comment:
This seems fine but we could follow the convention in [`NormalizerBase`](https://git.drupalcode.org/project/drupal/-/blob/9.4.x/core/modules/serialization/src/Normalizer/NormalizerBase.php#L96-100) with how things are referenced to be consistent.

Similarly, may want to expand the dependency to the entire Context entity and not just the tags?

```suggestion
        if ($context_config->getReaction($reaction->getPluginId()) && isset($context[CacheableNormalizerInterface::SERIALIZATION_CONTEXT_CACHEABILITY])) {
          $context['cacheability']->addCacheableDependency($context_config);
        };
```

If we do the above would need to add `use Drupal\serialization\Normalizer\CacheableNormalizerInterface;`.